spec/controllers/two_factor_authentication/backup_code_verification_controller_spec.rb

```diff
@@ -50,6 +50,27 @@
         expect(subject.user_session[TwoFactorAuthenticatable::NEED_AUTHENTICATION]).to eq false
       end
 
+      context 'with remember_device in the params' do
+        it 'saves an encrypted cookie' do
+          remember_device_cookie = instance_double(RememberDeviceCookie)
+          allow(remember_device_cookie).to receive(:to_json).and_return('asdf1234')
+          allow(RememberDeviceCookie).to receive(:new).and_return(remember_device_cookie)
```

Comment:
😭 my least favorite stub, not seeing a good alternative at the moment

Reply:
Yeah, this was copied from another test. I was trying to come up with a better way.

Reply:
Removed this and other instances of the stubbing to use the `RememberDeviceCookie` class methods directly in [7f37583](https://github.com/18F/identity-idp/pull/8835/commits/7f3758355a3e48edfb097ee9b9efc4b30a2f8114)